docker-compose.yml

```
@@ -287,11 +287,11 @@ services:
     restart: always
     logging: *default-logging
   worship-submissions-email-notification-service:
-    image: lblod/worship-submissions-email-notification-service:1.1.0
+    image: lblod/worship-submissions-email-notification-service:1.2.0
     environment:
       RUN_INTERVAL: "0 10 * * *"
       OUTBOX_FOLDER_URI: "http://data.lblod.info/id/mail-folders/2"
-      FROM_EMAIL_ADRESS: "Agentschap Binnenlands Bestuur Vlaanderen <noreply-binnenland@vlaanderen.be>"
+      FROM_EMAIL_ADDRESS: "Agentschap Binnenlands Bestuur Vlaanderen <noreply-binnenland@vlaanderen.be>"
       WORSHIP_DECISIONS_APP_BASEURL: "https://databankerediensten.lokaalbestuur.vlaanderen.be/"
```

` worship-submissions-email-notification-service` is missing the `restart` and `logging` env vars:
```suggestion
      WORSHIP_DECISIONS_APP_BASEURL: "https://databankerediensten.lokaalbestuur.vlaanderen.be/"
      restart: always
      logging: *default-logging
```

---

@Poltergeistz I forgot to indent them properly. `restart` and `logging` should be under the service itself:
```yml
worship-submissions-email-notification-service:
  image:
  environment:
    ENV_VAR_1:
    ENV_VAR_2:
  restart: always
  logging: *default-logging
```